Linear/Test/CanTS2D-fields-p2.xinp

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+
+<!-- 2D Cantilever beam with a tip shear load. !-->
+
+<simulation>
+
+  <geometry Lx="2.0" Ly="0.4">
+    <refine patch="1" v="1"/>
+    <raiseorder patch="1" u="1" v="1"/>
+    <refine patch="1" u="9"/>
+  </geometry>
+
+  <boundaryconditions>
+    <fixpoint patch="1" rx="0.0" ry="0.0" code="1"/>
+    <fixpoint patch="1" rx="0.0" ry="0.5" code="12"/>
+    <fixpoint patch="1" rx="0.0" ry="1.0" code="1"/>
+    <propertycodes>
+      <code value="1001">
+        <patch index="1" edge="1"/>
+      </code>
+      <code value="1002">
+        <patch index="1" edge="1"/>
+        <patch index="1" edge="2"/>
+      </code>
+    </propertycodes>
+    <neumann code="1001" direction="1" type="expression">
+      F0=-1000000; L=2; H=0.4; I=H*H*H/12; Y=y/H-0.5;
+     -F0*(L*H/I)*Y
+    </neumann>
+    <neumann code="1002" direction="2" type="expression">
+      F0=-1000000; L=2; H=0.4; I=H*H*H/12; Y=y/H-0.5;
+      F0*(H*H/I)*(0.5-x/L)*(0.25-Y*Y)
+    </neumann>
+  </boundaryconditions>
+
+  <elasticity>
+    <isotropic E="2.068e11" nu="0.29" rho="7820.0"/>
+    <anasol type="fields" file="CanTS2D-p3" level="1">
```

Comment:
probably want level 0.

Reply:
tried that first, but made no difference, the problem is that the xml object is empty such that the loop in AnaSol.C:l285     `for (const XMLWriter::Entry& it : xml.getEntries())` does nothing